Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ruby:2.4-alpine
+
+RUN apk --update add grep
+
+WORKDIR /usr/src/app
+COPY . /usr/src/app
```

Comment:
We usually copy the Gemfile/.lock files and bundle first so that we can keep a cached layer including all gems while working on other files:

```dockerfile
COPY Gemfile* ./
RUN gem install -g
COPY . ./
```

Also, when specifying a WORKDIR, you can use relative destinations within subsequent commands.